### DISCUSSION TITLE
l'essentiel du cours du troisième  séance

### DISCUSSION THREAD

**Discussion by mouna27a** [solid-de/learn-java#15] [Week3]
**Java prend en charge les conditions logiques suivantes:**

1. a == b : A égale à B
2. a != b : A différent de B
3. a < b : A inférieur de B
4. a <= b : A inférieur ou égal à B
5. a > b : A supérieur à B
6. a >= b : A supérieur ou égal à B

 **Structure conditionnelle if-else en Java:**

-Si l'évaluation de l'expression booléenne test donne true (la valeur 'vrai'), alors l'instruction 1 est exécutée; sinon l'expression booléenne a donné false (la valeur 'faux') et l'instruction 2 est exécutée.

**Syntaxe:**
•	if ( condition ) {
•		instructions à exécuter si la condition est true ;
•	}else{
•	        instructions à exécuter si condition est false
•	}
**exemple:**
public class Main {
  public static void main(String[] args) {
    int time = 20;
    if (time < 18) {
      System.out.println("Good day.");
    } else {
      System.out.println("Good evening.");
    }  
  }
}
**resultat:**
Good evening.

**La boucle for: répétition inconditionnelle:**
-La répétition inconditionnelle consiste à répéter n fois le contenu de la boucle: on connait à l'avance le nombre de répétitions.
**exemple : afficher 5 fois  "Bonjour":**
System.out.println("Bonjour");
System.out.println("Bonjour");
System.out.println("Bonjour");
System.out.println("Bonjour");
System.out.println("Bonjour");
 **résultat:**
Bonjour
Bonjour
Bonjour
Bonjour
Bonjour
**Ce code peut-être écrit de façon beaucoup plus efficace avec une boucle:**
for (int i=0; i<5; ++i){
    System.out.println("Bonjour");
}
**résultat:**
Bonjour
Bonjour
Bonjour
Bonjour
Bonjour

**Comment by MnaouerA** [comment]
S'il vous plaît, la boucle répétition inconditionnel
L'instruction (for (int i=0; i<5; ++i){). Je l'ai pas compris

**Reply by raissi** [reply]
Bonjour,
Si vous parlez Arabe tunisien, voici un cours de 8 séances:
https://youtube.com/playlist?list=PLZc_bilpXSF5hGYna7mXCZ7vEqe58eOth&si=8_m4DQD63_LwClBx
